include/adapters/pulseaudio.hpp

```diff
@@ -74,6 +74,8 @@ class pulseaudio {
     string spec_s_name;
     string s_name;
     uint32_t m_index{0};
+
+    pa_volume_t max_volume = PA_VOLUME_UI_MAX;
```

Comment:
Could you please use brace initialization here?

Reply:
Yeah, was in a hurry this morning...